codegen/files_to_generate/emosqp_mex.c

```diff
@@ -299,6 +299,34 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
         return;
     }
 
+    // update warm_start
+    if (!strcmp("update_warm_start", cmd)) {
```

Comment:
```suggestion
    if (!strcmp("warm_start", cmd)) {
```

The class wrapper doesn't call `update_warm_start` actually, it will call `warm_start` when it wants to update both vectors.

Reply:
Actually, I may have gotten confused here because I didn't see this was the `emosqp` file, so this change probably isn't actually needed (although we need to document this now in https://github.com/oxfordcontrol/osqp/blob/master/docs/codegen/matlab.rst as well).